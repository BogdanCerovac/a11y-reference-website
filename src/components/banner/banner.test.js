import React from "react"
import renderer from "react-test-renderer"
import Banner from "./banner"

describe("Banner", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Banner>My banner</Banner>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
