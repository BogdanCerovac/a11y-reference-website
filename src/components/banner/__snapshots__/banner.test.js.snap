// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner renders correctly 1`] = `
<div
  className="banner"
>
  My banner
</div>
`;
